js/ViewPager.js

```diff
@@ -163,7 +163,7 @@ class ViewPager extends React.Component<ViewPagerProps> {
    * `ViewPager` wants to prevent a child `View` from becoming responder on a move
    * See https://github.com/react-native-community/react-native-viewpager/issues/164
    */
-  _onMoveShouldSetResponderCapture = () => {
+  _onMoveShouldSetResponderCapture = () => true;
     return true;
```

Comment:
need to remove this now 🕺

Reply:
I cannot touch anything if it returns `true`
Why always `true` instead of `this.isScrolling`?

Reply:
If I return this.isScrolling => Can scroll but sometimes I have accidental touches.
If I return false => It always causes accidental touches.